app/services/jbpm_process_service.rb

```diff
@@ -0,0 +1,37 @@
+class JbpmProcessService
+  attr_accessor :request
+
+  def initialize(request)
+    self.request = request
+  end
+
+  def start
+    options = template.process_setting
+    Kie::Service.call(KieClient::ProcessInstancesBPMApi, options) do |bpm|
+      bpm.start_process(options['container'], options['process'], :body => process_options)
+    end
+  end
+
+  def signal(decision)
```

Comment:
In our case, we just send `signal` and let approval process go, may not need `signal_option` here, or keep it but provide a default value.

Reply:
We do need to send the decision to bpm. BPM will decide whether to skip the next group.

Reply:
👍